presidio-analyzer/presidio_analyzer/predefined_recognizers/crypto_recognizer.py

```diff
@@ -1,35 +1,49 @@
 from hashlib import sha256
-from presidio_analyzer import Pattern
-from presidio_analyzer import PatternRecognizer
+from presidio_analyzer import Pattern, PatternRecognizer
 
 # Copied from:
 # http://rosettacode.org/wiki/Bitcoin/address_validation#Python
-REGEX = r'\b[13][a-km-zA-HJ-NP-Z1-9]{26,33}\b'
-CONTEXT = ["wallet", "btc", "bitcoin", "crypto"]
 
 
 class CryptoRecognizer(PatternRecognizer):
     """
     Recognizes common crypto account numbers using regex + checksum
     """
 
-    def __init__(self):
-        patterns = [Pattern('Crypto (Medium)', REGEX, 0.5)]
-        super().__init__(supported_entity="CRYPTO", patterns=patterns,
-                         context=CONTEXT)
+    PATTERN_GROUPS = [
+        ("Crypto (Medium)", r"\b[13][a-km-zA-HJ-NP-Z1-9]{26,33}\b", 0.5),
```

Comment:
I use it in the [`__init__`](https://github.com/dhpollack/presidio/blob/dhp/allow_multiple_langs/presidio-analyzer/presidio_analyzer/nlp_engine/__init__.py) of the nlp_engine, so that we know what engines are available.  It also makes it easier to add new engines if someone wanted to do that by having a consistent interface.